### ISSUE TITLE
Move automatically when there is only 1 valid move?

### ISSUE THREAD

**Issue by tb148** [rossjrw/ur#1123]
When there's only 1 valid move, there are no decisions to make, so this move can be done automatically.

**Comment by rossjrw** [comment]
Yeah, that's a good idea. I will look into it, thanks for the suggestion!

**Comment by suprstarrd** [comment]
In parity with #1293, a notice should probably be accompanied with *all* automatic moves.
